TestCases/compliance-level-3/0050-feel-abs-function/0050-feel-abs-function.dmn

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0050-feel-abs-function" name="0050-feel-abs-function" id="_i9fboPUUEeesLuP4RHs4vA" xmlns="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <description>FEEL built-in function 'abs(number)' in category numeric functions</description>
+    <decision name="decision001" id="_decision001">
+        <description>Tests FEEL expression: 'abs(1)' and expects result: '1 (number)'</description>
+        <question>Result of FEEL expression 'abs(1)'?</question>
+        <allowedAnswers>1 (number)</allowedAnswers>
+        <variable typeRef="number" name="decision001"/>
+        <literalExpression>
+            <text>abs(1)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision002" id="_decision002">
+        <description>Tests FEEL expression: 'abs(-1)' and expects result: '-1 (number)'</description>
```

Comment:
from `<description>Tests FEEL expression: 'abs(-1)' and expects result: '-1 (number)'</description>`
to `<description>Tests FEEL expression: 'abs(-1)' and expects result: '1 (number)'</description>`

Reply:
@tarilabs amended.